src/parsley/validator_registry.js

```diff
@@ -359,7 +359,16 @@ ValidatorRegistry.prototype = {
           return value === refOrValue;
       },
       priority: 256
-    }
+    },
+    euvatin: {
```

Comment:
Check how `integer` is implemented. You don't need an actual function.

Reply:
I tried but it fails integration tests with an obscure error.